scripts/sell_unused_slots.py

```diff
@@ -151,9 +151,17 @@ def main(argv):
     if is_active is False:
         log.info('No slots are currently reserved.')
     elif is_active is True:
-        monitor = ReindexDetector()
-        if not monitor.is_reindex_active():
-            reservation.deactivate()
+        min_reservation_age = 30 * 60
+        now = datetime.utcnow().timestamp()
```

Comment:
What's the difference between this and `time.time()`? If there is none, I'd prefer the latter.

Reply:
`time.time()` is in the current timezone and thus, on my computer (in UTC-7) differs from `datetime.datetime.utcnow()` by 7 hours:

```
>>> (time.time() - datetime.datetime.utcnow().timestamp())/60/60
-7.0000000016556845
```

Reply:
The reason you're seeing a difference of seven hours is not because `time.time()` is in local time. It is because `.utcnow()` returns a naíve `datetime` object which `.timestamp` converts to a timestamp in local time. The correct way to get a UTC timestamp using `datetime` is via `.now(timezone.utc)` as specified in the big red warning in the [documentation] on `.utcnow()`](https://docs.python.org/3.8/library/datetime.html#datetime.datetime.utcnow).

```python
time.time() - datetime.datetime.now(datetime.timezone.utc).timestamp()
-2.86102294921875e-06
```

I have not seen a Unix on which `time.time()` doesn't return UTC so I think we should stick with that since that's what we have been doing.